ctmc_lectures/generators.md

```diff
@@ -359,7 +359,7 @@ Indeed, why should the limit exist, given that $C_0$ semigroups are not
 required to be differentiable?
 
 The other problem is that, even though the limit exists, the linear operator
-$A$ is not bounded (i.e., not an element of $\linop$), so 
+$A$ may be not bounded (i.e., not an element of $\linop$), so 
```

Comment:
Good idea to change this @shlff .  Could you please replace "may be not bounded" with "might be unbounded", and "so" with "in which case".  Thanks.

Reply:
Thanks @jstac for your better ideas. I modified the commit according to your advice.